### ISSUE TITLE
tracer 0.1.0 is included with ruby but doesn't exist on rubygems.org

### ISSUE THREAD

**Issue by deivid-rodriguez** [ruby/tracer#1]
On a fresh ruby 2.6.1 installation, we can see `tracer 0.1.0` is a default gem

```shell
$ gem list tracer

*** LOCAL GEMS ***

tracer (default: 0.1.0)
```

However, this version does not exist on rubygems.org

```shell
gem search ^tracer$ --all

*** REMOTE GEMS ***

tracer (0.0.1)
```

`tracer` 0.1.0 should probably be pushed to rubygems.org.

Same thing happens with the `prime` gem, but I didn't report it because issues are closed on https://github.com/ruby/prime.

**Comment by hsbt** [comment]
Hi, @deivid-rodriguez 

I send a transfer request to namespace holders of `tracer` and `prime`. But I have no response yet. I need to enforce to transfer them to ruby organization. But It's a low priority now.

**Comment by deivid-rodriguez** [comment]
I understand. So until that happens, we can't make any releases of these gems, right?

**Comment by hsbt** [comment]
Yes.

**Comment by headius** [comment]
This also holds up sourcing tracer from the gem in JRuby 9.3. I will restore tracer from stdlib directly for now.

**Comment by headius** [comment]
jruby/jruby#5576

**Comment by deivid-rodriguez** [comment]
The prime case was fixed recently and @hsbt got ownership due to current maintainers unavailability, so since https://github.com/rubygems/rubygems.org/issues/2259 was closed, maybe ownership for `tracer` was granted too?

**Comment by hsbt** [comment]
I'm working to ship them now.

**Comment by hsbt** [comment]
@headius `tracer-0.1.0` has been released today. It's the same code that is shipped with Ruby 2.6.

**Comment by headius** [comment]
@hsbt Thank you very much!
